app/code/Magento/GraphQl/Controller/GraphQl.php

```diff
@@ -128,6 +136,13 @@ public function dispatch(RequestInterface $request) : ResponseInterface
             $result['errors'][] = $this->graphQlError->create($error);
             $statusCode = ExceptionFormatter::HTTP_GRAPH_QL_SCHEMA_ERROR_STATUS;
         }
+
+        if (array_key_exists('errors',$result) && !empty($result['errors'])) {
+            foreach ($result['errors'] as $error) {
+                $this->logger->warning($error['message']);
```

Comment:
1. How would it look in the log? Will it be possible to trace back to the original class where exception was thrown?
2. In case of internal errors we need to show the client log ID, so that it can be reported to the support for further investigation, the same ID must be specified in the log file. Please see how it is done in REST implementation